src/apps/chatter/components/room-password/room-password.js

```diff
@@ -9,8 +9,8 @@ export const RoomPassword = ({
   handleSubmitPassword,
   isChecking,
   showWrongPasswordMessage
-}) => (
-  <Modal className='styled-modal small'>
+}) => {
+  return <Modal className='styled-modal small'>
```

Comment:
This change doesn't seem necessary and is arguable a less desirable style.

```
const Comp = (props) =>
  <div>
    ...
  </div>
```

or 

```
function Comp (props) {
  ...
  return (
    <div>
      ...
    </div>
  )
}
```

Are the two preferred options ✨

Reply:
Agreed